docs/content/docs/concepts/transactions.mdx

```diff
@@ -3,4 +3,472 @@ title: Transactions
+
+```ts twoslash
+import {
+    TransactionMessage,
+    TransactionMessageWithFeePayer,
+    TransactionMessageWithSigners,
+} from '@solana/kit';
+const transactionMessage = null as unknown as TransactionMessage &
+    TransactionMessageWithFeePayer &
+    TransactionMessageWithSigners;
+// ---cut-before---
+import { compileTransaction } from '@solana/kit';
+
+const transaction = compileTransaction(transactionMessage);
+```
+
+This produces an unsigned transaction. Follow the [instructions for signing transactions with `CryptoKeyPairs`](#signing-transactions) to sign it.
+
+## Signing transactions
+
+Wherever you have a `Transaction` instead of a `TransactionMessage` you can add or replace a signature using the [`signTransaction`](/api/functions/signTransaction) method and one or more `CryptoKeyPairs`.
+
+```ts twoslash
+import { Transaction, TransactionWithLifetime } from '@solana/kit';
+const keyPair = null as unknown as CryptoKeyPair;
+const transaction = null as unknown as Transaction & TransactionWithLifetime;
+// ---cut-before---
+import {
+    SOLANA_ERROR__TRANSACTION__SIGNATURES_MISSING,
+    isSolanaError,
+    signTransaction,
+} from '@solana/kit';
+
+try {
+    const fullySignedTransaction = await signTransaction([keyPair], transaction);
+} catch (e) {
+    if (isSolanaError(e, SOLANA_ERROR__TRANSACTION__SIGNATURES_MISSING)) {
+        console.error('Missing signers for the following addresses:', e.context.addresses);
+    } else {
+        throw e;
+    }
+}
+```
+
+<Callout type="warn">
+    This function will throw if the resultant transaction is missing a signature for one of the
+    transaction's required signers. To partially sign a transaction, use the
+    [`partiallySignTransaction`](/api/functions/partiallySignTransaction) method.
+</Callout>
+
+## Serializing transactions
+
+If you would like to send a transaction to the network manually, you must first serialize it in a particular way. The [`Base64EncodedWireTransaction`](/api/type-aliases/Base64EncodedWireTransaction) represents the wire format of a transaction as a base64-encoded string.
+
+Given a transaction, the [`getBase64EncodedWireTransaction`](/api/functions/getBase64EncodedWireTransaction) method returns the transaction as a string that conforms to the `Base64EncodedWireTransaction` type.
+
+```ts twoslash
+import { Transaction, Rpc, SendTransactionApi } from '@solana/kit';
+const rpc = null as unknown as Rpc<SendTransactionApi>;
+const signedTransaction = null as unknown as Transaction;
+// ---cut-before---
+import { getBase64EncodedWireTransaction, signTransaction } from '@solana/kit';
+
+const serializedTransaction = getBase64EncodedWireTransaction(signedTransaction);
+const signature = await rpc.sendTransaction(serializedTransaction, { encoding: 'base64' }).send();
+```
```

Comment:
I think here we should link to the send and confirm helpers for discoverability. If a reader starts with this transaction guide, they will go back to their code thinking `rpc.sendTransaction` is the recommended way to send them using Kit. A simple callout with a link to `/docs/getting-started/send-transaction` should suffice.